mon-pix/app/components/challenge-statement.js

```diff
@@ -21,7 +22,18 @@ export default class ChallengeStatement extends Component {
     if (!instruction) {
       return null;
     }
-    return instruction.replace('${EMAIL}', this._formattedEmailForInstruction());
+    const externalLinkRegex = /(\[(.*?)\]\((.*?)\))+/;
+    const formattedInstruction = instruction.replace('${EMAIL}', this._formattedEmailForInstruction());
+    return formattedInstruction.replace(externalLinkRegex, this._insertLinkTitle.bind(this));
```

Comment:
```suggestion
    return formattedInstruction.replaceAll(externalLinkRegex, this._insertLinkTitle.bind(this));
```

Sinon, seul la première occurrence est remplacé. 
Dans le cas de l'épreuve recxDBTrZbLTPiaDi, seul le premier lien a ce title.